documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -1247,6 +1247,9 @@ https://kafka.apache.org/[Apache Kafka] is a popular open-source platform for di
 This means that all Kafka producer https://kafka.apache.org/documentation/#producerconfigs[configuration properties] are passed to the producer with the prefix removed.
 At least `bootstrap.servers`, `key.serializer` and `value.serializer` properties must be provided. The `topic` is set by {prodname}.
 
+|[[kafka-producer-wait-message-delivery-timeout-ms]]<<kafka-producer-wait-message-delivery-timeout-ms, `debezium.sink.kafka.wait.message.delivery.timeout.ms`>>
+|`30000`
+|The max wait time until request completes and returns the metadata for the record or throw any exception that occurred while sending the record to Apache Kafka. Set to `0` to wait indefinitely.
```

Comment:
```suggestion
|The maximum time, in milliseconds, that the server waits for a request to complete and return metadata for a record, or to throw an exception when a failure occurs when sending a record to Apache Kafka. 
Set the value to `0` to disable the timeout.
```
Regarding the part about throwing an exception, does the specified timeout value determine how long the server waits to generate an exception after Kafka reports that an operation failed? 
Or does the timeout govern how long the server waits for Kafka to report that an operation failed?

Reply:
> Or does the timeout govern how long the server waits for Kafka to report that an operation failed?
It's this one. If Kafka won't respond during the timeout period, it will raise an exception.